pinot-plugins/pinot-stream-ingestion/pinot-pulsar/src/main/java/org/apache/pinot/plugin/stream/pulsar/PulsarConfig.java

```diff
@@ -46,6 +46,8 @@ public class PulsarConfig {
   private final String _authenticationToken;
   private final String _tlsTrustCertsFilePath;
   private final boolean _enableKeyValueStitch;
```

Comment:
I'm not familiar with the process for deprecating a config param. Should I just add the `@Deprecated` annotation?

Reply:
Yeah. You can add `@Deprecated(since = "v0.13.* since pulsar support record key extraction")` 
Might have to update documentation (if it exists) in the website